### ISSUE TITLE
Step by step instructions for beginner

### ISSUE THREAD

**Issue by jamesbrysmart** [headmelted/codebuilds#39]
Hi, just before we get started, I am an absolute beginner with all of this so please be patient. I'm currently trying to download software (Git and VS code) for a software development course I am soon to start, onto my Chromebook, an Asus C100PA with ARM processor.

I've had a play around on my own for the last day or so and have learnt a lot already about Croutons/ Linux etc, but am finally totally stuck. From what (I think) I've learnt, the ARM processor has a compatibility issue with VS code so I ran the relevant install command, which seems to have worked as it says on the terminal 'Visual Studio Code install complete'.

But I don't know what to do now? My previous attempts with chroots I'd input 'sudo enter-chroot startxfce4' and that would get me in, though similar attempts now mean the command line has changed to (code-oss-chroot)jamesbryant@localhost:~$ but that's it, and now all attempts at commands are being met with 'Command not found'. I can't see VS code anywhere, or have any idea how to find it.

Help please? Or even links to Linux for beginners which would help me get my head around it? Have read a fair bit already but not sure if I'm making a mistake or there's compatibility problems and I'm wasting my time.

Thanks so much - James

**Comment by magic-k** [comment]
`sudo enter-chroot -n code-oss-chroot`
to enter the chroot. The command is 
`code-oss`
Another way is
`sudo startxiwi -n code-oss-chroot /usr/bin/code-oss`
This starts code in a window. If you add -.b ist detaches from the chroot.

**Comment by jamesbrysmart** [comment]
Thanks for this - I've followed your instructions, though I just got this error?

ERROR: Connection failed (code: -2). 

The only thing I've done since is uninstall my xfce4 chroot, and install a Gnome chroot, which also doesn't appear to be working.

Appreciate your help, thanks

**Comment by magic-k** [comment]
Do you have the crouton extension installed?

**Comment by jamesbrysmart** [comment]
Yes - but it won't connect. Keeps trying and saying disconnected
